docker-compose.yml

```diff
@@ -2,19 +2,13 @@ version: "2"
 
 services:
   magneticod:
-    build:
-      image: boramalper/magneticod
-#      context: .
-#      dockerfile: Dockerfile.magneticod
+    image: boramalper/magneticod
     volumes:
       - magneticod-data:/root/.local/share/magneticod
       - magneticod-config:/root/.config/magneticod
 
   magneticow:
-    build:
-      image: boramalper/magneticow
-#      context: .
-#      dockerfile: Dockerfile.magneticow
+    image: boramalper/magneticow
```

Comment:
[`build`](https://docs.docker.com/compose/compose-file/#build) and [`image`](https://docs.docker.com/compose/compose-file/#image) sections could be used together.

See https://docs.docker.com/compose/compose-file/#image :

> If the image does not exist, Compose attempts to pull it, unless you have also specified build, in which case it builds it using the specified options and tags it with the specified tag.

Reply:
Yes it does, but putting `image` in `build` lead to the error as described in https://github.com/boramalper/magnetico/issues/232

I removed the `build` to use the image from docker hub by default, and we still can build our own image with the already existing `make image` command. docker-compose will then use them instead of the docker hub ones

Reply:
Yes. That's because `image` and `build` section must be on the same level of configuration and not nested in each other. So you can just fix the error in the original `docker-compose.yml` instead of removing `build` section.